examples/AspNetCore/RoutingSample/Startup.cs

```diff
@@ -59,13 +59,13 @@ public Startup(IConfiguration configuration)
         /// <param name="services">Service Collection.</param>
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddDaprClient();
-
-            services.AddSingleton(new JsonSerializerOptions()
-            {
-                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
-                PropertyNameCaseInsensitive = true,
-            });
+            services.AddDaprClient(builder =>
+              builder.UseJsonSerializationOptions(
+                  new JsonSerializerOptions()
+                  {
+                      PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                      PropertyNameCaseInsensitive = true,
+                  }));
```

Comment:
This actually breaks the example. You can see on line 77, the `JsonSerializerOptions` needs to be a directly registered service.

What was your intent here?

Reply:
It seems that I made a mistake,  I thought it used a `JsonSerializerOptions` object with ` DaprClient`